WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -930,6 +936,17 @@ private void showNewPostsBar() {
         AniUtils.startAnimation(mNewPostsBar, R.anim.reader_top_bar_in);
         mNewPostsBar.setVisibility(View.VISIBLE);
 
+        // hide the bar when the recycler is scrolled
+        mRecyclerView.addOnScrollListener(new RecyclerView.OnScrollListener() {
+            @Override
+            public void onScrolled(RecyclerView recyclerView, int dx, int dy) {
+                super.onScrolled(recyclerView, dx, dy);
+                if (dy >= mHideNewPostsBarOnScrollOffset) {
+                    hideNewPostsBar();
+                }
```

Comment:
I think the `onScrolled` callback could be called repeatedly and thus `hideNewPostsBar` would be called repeatedly too (even though it won't do anything if an animation is being performed, as I could see inside that method). 
Also, a new listener is created and then added each time `showNewPostsBar` is called, leading to potentially having several onScrolled listeners living there. I know this seems to be covered by he `clearOnScrollListeners` call in `hideNewPostsBar`, but it would be a potential bug in the case someone adds a new onScrolled listener for a different purpose in the future.

It might be a good thing to either have one instance of the scroll listener and have a reference in a class member (then add it to the recycler view in `onStart` then call `removeOnScrollListener` in `onStop`) , or continue with the anonymous instance but have itself removed inside this fx once the hideNewPostsBar() has been called.